### PR TITLE
Added Cloud SQL postgres + authorized network example for inclusion on CGC

### PR DESCRIPTION
**Pull request by betsy-lichtenberg**
Added example for inclusion on the following pages:

https://cloud.google.com/sql/docs/postgres/authorize-networks

```release-note:none
```

### CODE REVIEW

File: mmv1/templates/terraform/examples/sql_postgres_instance_authorized_network.tf.erb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    tier = "db-custom-2-7680"
+    ip_configuration {
+      authorized_networks {
+        name = "Network Name"
+        value = "192.0.2.0/24"
+        expiration_time = "3021-11-15T16:19:00.094Z"
+      }
+    }
+  }
+  deletion_protection =  "false"
```
Comment (megan07) [inline mmv1/templates/terraform/examples/sql_postgres_instance_authorized_network.tf.erb:16]: I think we want to use a variable to define this to differentiate the docs from the tests, similar to [here](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/templates/terraform/examples/sql_database_instance_postgres.tf.erb#L9).

**Review by megan07 — CHANGES_REQUESTED** [review]

File: mmv1/templates/terraform/examples/sql_postgres_instance_authorized_network.tf.erb (hunk 1 above)
Comment (betsy-lichtenberg) [reply mmv1/templates/terraform/examples/sql_postgres_instance_authorized_network.tf.erb]: done

**Comment by megan07** [comment]
Hi @betsy-lichtenberg ! Would you mind rebasing off main please? I just want to pull in those changes before we run the tests again. Thanks!

**Comment by megan07** [comment]
Sorry @betsy-lichtenberg, I'm still seeing some merge conflicts, I was hoping doing a rebase would help did you run `git pull --rebase upstream main`? Sorry about this!

**Comment by betsy-lichtenberg** [comment]
I guess I was too ambition trying to have multiple PRs for the same file. I'll close this and try again later.

**Pull request closed without merging**